x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/response/rules_schema.ts

```diff
@@ -139,11 +143,30 @@ export const rulesSchema = new t.Type<
   'RulesSchema',
   (input: unknown): input is RulesWithoutTypeDependentsSchema => isObject(input),
   (input): Either<t.Errors, RulesWithoutTypeDependentsSchema> => {
-    return checkTypeDependents(input);
+    const output = checkTypeDependents(input);
+    if (!hasListsFeature()) {
+      // TODO: Remove this after the lists feature is an accepted feature for a particular release
```

Comment:
Just an idea of maybe adding some easily searchable key to these comments like `// TODO (LIST-FEATURE): ...` so when the time comes we can easily find all the TODOs? That may just be a me thing :)

Reply:
This is good advice. Thank you.

Updated all of it to have:
// TODO: (LIST-FEATURE) ...any comments...

So it will be easier to find all the spots when we remove them.